### PR TITLE
Add and export ranefinfotable.  Add tests.

### PR DESCRIPTION
**Pull request by dmbates**

### CODE REVIEW

File: src/caterpillar.jl [hunk 1]
```diff
@@ -52,6 +52,23 @@ end
...
+"""
+function ranefinfotable(ri::RanefInfo)
+    cnames, levels = ri.cnames, ri.levels
+    k = length(cnames)
+    l = length(levels)
+    (;
+        cname=repeat(cnames; inner=l),
+        level=repeat(levels; outer=k),
+        μ=vec(ri.ranef),
+        σ=vec(ri.stddev),
```
Comment (palday) [inline src/caterpillar.jl:68]: Naming is hard....

Given that these are only means for LMM, is there another name that would be short and sweet?

Or maybe just add a note to the doc string about the fields and their meaning?

**Review by palday — COMMENTED** [review]
Minor documentation/naming question.

If you do a patchbump, I can also immediately tag a release.

File: src/caterpillar.jl (hunk 1 above)
Comment (dmbates) [reply src/caterpillar.jl:68]: Do you think `cmode` and `cstddev` would be better than `μ` and `σ`?

The phrase "conditional mode" came from me being pedantic.  Technically "conditional standard deviation" applies under the same circumstances as "conditional mean" - i.e. only to LMMs.  But I wouldn't know what to call it for a GLMM so I sort of punted there. :smiley:

Comment (palday) [reply src/caterpillar.jl:68]: I like the `c` vocabulary. It also makes it clear these standard deviations are not the standard deviations from VarCorr.

Comment (dmbates) [reply src/caterpillar.jl:68]: In that case I think I will rename `cname` to `name` to avoid confusion with the `c` prefix.

**Comment by dmbates** [comment]
I noticed that the `Project.toml` file lists 
```
MixedModels = "4.1"
```
in the compat section.  Does it need to be that specific? If "4.0" isn't good then it should be extended to
```
MixedModels = "4.1, 4.2, 4.3, 4.4"
```
shouldn't it?

BTW, I see you changed the version of MixedModels.jl to '4.4.0' in `Project.toml` but have not released it.  Is that intentional?

**Comment by palday** [comment]
I think we're actually taking advantage of something in this package that requires MM 4.1 (maybe the evaluation of `condVar` for only a single blocking variable?).  

I hadn't released MixedModels.jl 4.4 yet because we were adding in enough stuff that I thought we might have even more stuff to add. But I can release it today :smile:

**Review by palday — APPROVED** [review]

**Pull request merged**